src/rules/whitespaceRule.ts

```diff
@@ -192,6 +196,13 @@ function walk(ctx: Lint.WalkContext<Options>) {
                 if (options.decl && initializer !== undefined) {
                     checkForTrailingWhitespace((type !== undefined ? type :  name).getEnd());
                 }
+                break;
+
+            case ts.SyntaxKind.SpreadAssignment:
```

Comment:
This also checks `foo(...bar)`, which is correct if this option checks rest *and* spread. 
It would also need to check `BindingElement` and `Parameter` where `dotDotDotToken` is not undefined and `SpreadElement`.